src/js/pages/Department.js

```diff
@@ -13,70 +13,60 @@ import FormFeedback from 'js/page_sections/FormFeedback';
 import Service311 from 'js/page_sections/Service311';
 import departmentPageQuery from 'js/queries/departmentPageQuery';
```

Comment:
Still minor, can we kill these single page query imports (in the Department and Topic component files) since we're doing all page queries at the top level?